### PR TITLE
Feature suse 15

### PR DESCRIPTION
**Pull request by eradot4027**
#### Description:

- Continued adding support for SLE15.  Created a profile to begin reflecting the SLE15 Hardening guide's suggested configuration. Modified linux_os rules for sle15 support.

#### Rationale:

- Adding support in the SSG for SLE15

### CODE REVIEW

**Comment by redhatrises** [comment]
@eradot4027 please undo changes to `shared/references/disa-stig-sles11-v1r7-xccdf-manual.xml` this file is not to be modified. Thanks.

**Comment by redhatrises** [comment]
@openscap-ci test this please

**Comment by redhatrises** [comment]
@eradot4027 also... thinking about it. It might also makes sense to merge all the rules into the `standard.profile` rather than creating the `suse-hardening-guide.profile`. Basically the `standard.profile` would be the profile that contains suse's hardening guidance.

**Comment by redhatrises** [comment]
@openscap-ci add to whitelist

File: linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_interactive_users/rule.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: ol7,ol8,rhel7,rhel8,rhv4,wrlinux1019
+prodtype: ol7,ol8,rhel7,rhel8,rhv4,sles15,wrlinux1019
```
Comment (redhatrises) [inline linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_interactive_users/rule.yml]: s/sles/sle/g

File: linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_bashrc/rule.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: ol7,ol8,rhel6,rhel7,rhel8
+prodtype: ol7,ol8,rhel6,rhel7,rhel8,sles15
```
Comment (redhatrises) [inline linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_bashrc/rule.yml]: s/sles/sle/g

File: linux_os/guide/system/accounts/accounts-session/accounts_tmout/rule.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: fedora,ol7,ol8,rhel6,rhel7,rhel8,rhv4,wrlinux1019
+prodtype: fedora,ol7,ol8,rhel6,rhel7,rhel8,rhv4,sles15,wrlinux1019
```
Comment (redhatrises) [inline linux_os/guide/system/accounts/accounts-session/accounts_tmout/rule.yml]: s/sles/sle/g

File: sle15/profiles/standard.profile [hunk 4]
```diff
@@ -4,8 +4,82 @@ title: 'Standard System Security Profile for SUSE Linux Enterprise 15'
 
 description: |-
     This profile contains rules to ensure standard security baseline
-    of a SUSE Linux Enterprise 15 system. Regardless of your system's workload
-    all of these checks should pass.
+    of a SUSE Linux Enterprise 15 system. Includes recommendations based on SUSE
+    Hardening Guide
```
Comment (redhatrises) [inline sle15/profiles/standard.profile]: Let's update the description to the following:

```
    This profile contains rules to ensure standard security baseline
    of a SUSE Linux Enterprise 15 system based off of the SUSE 
    Hardening Guide. Regardless of your system's workload all of
    these checks should pass.
```

File: linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_interactive_users/rule.yml (hunk 1 above)
Comment (eradot4027) [reply linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_interactive_users/rule.yml]: Forgot to push after I fixed it last night.  I apologize

File: linux_os/guide/system/accounts/accounts-session/accounts_tmout/rule.yml (hunk 3 above)
Comment (eradot4027) [reply linux_os/guide/system/accounts/accounts-session/accounts_tmout/rule.yml]: Forgot to push after I fixed it last night.  I apologize

File: sle15/profiles/standard.profile (hunk 4 above)
Comment (eradot4027) [reply sle15/profiles/standard.profile]: Complete

File: linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_bashrc/rule.yml (hunk 2 above)
Comment (eradot4027) [reply linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_bashrc/rule.yml]: Complete

**Comment by redhatrises** [comment]
@openscap-ci test this please

**Comment by redhatrises** [comment]
Thanks! Ack.

**Comment by eradot4027** [comment]
I will delete it and start a new branch for any further modifications.  Thanks

**Pull request merged**